### PR TITLE
[visualization] fix visualization of ACK messages

### PR DESCRIPTION
**Pull request by simonlingoogle**
ACK messages was visualized same as broadcast messages, which was a bug. 

This PR fixes the visualization of ACK messages to use green triangles. 

Related PR: #23 turns off visualization of ACK messages by default. 

Note: `web/site/bindata.go` is a generated file.

### CODE REVIEW

File: web/site/js/vis/PixiVisualizer.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ import ActionBar from "./ActionBar";
 import {Text} from "./wrapper";
 import {MAX_SPEED, PAUSE_SPEED} from "./consts";
 import Node from "./Node"
-import {BroadcastMessage, UnicastMessage} from "./message";
+import {BroadcastMessage, UnicastMessage, AckMessage} from "./message";
```
Comment (wgtdkp) [inline web/site/js/vis/PixiVisualizer.js:33]: sort those names?

File: web/site/js/vis/PixiVisualizer.js [hunk 2]
```diff
@@ -235,7 +235,12 @@ export default class PixiVisualizer extends VObject {
         }
 
         let src = this.nodes[srcId];
-        if (dstId == -1) {
+
+        let frameType = mvInfo.getFrameControl() & 0x0007;
+        if (frameType === 2) {
```
Comment (wgtdkp) [inline web/site/js/vis/PixiVisualizer.js:240]: Better define enumerations for `Frame Type`.

File: web/site/js/vis/PixiVisualizer.js (hunk 1 above)
Comment (simonlingoogle) [reply web/site/js/vis/PixiVisualizer.js:33]: let's wait until some javascript formatting tools are adopted.

File: web/site/js/vis/PixiVisualizer.js (hunk 2 above)
Comment (simonlingoogle) [reply web/site/js/vis/PixiVisualizer.js]: hardcoding removed

File: web/site/js/vis/PixiVisualizer.js (hunk 1 above)
Comment (simonlingoogle) [reply web/site/js/vis/PixiVisualizer.js]: reordered

File: web/site/js/vis/consts.js [hunk 3]
```diff
@@ -25,4 +25,7 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 export const PAUSE_SPEED = 0;
-export const MAX_SPEED = 1000000;
\ No newline at end of file
+export const MAX_SPEED = 1000000;
+
+export const FRAME_CONTROL_MASK_FRAME_TYPE = 0x7;
+export const FRAME_TYPE_ACK = 2;
```
Comment (simonlingoogle) [inline web/site/js/vis/consts.js:31]:
```suggestion
export const FRAME_TYPE_ACK = 2;

```

File: web/site/js/vis/message.js [hunk 4]
```diff
@@ -126,3 +126,52 @@ export class UnicastMessage extends VObject {
...
+        this._root = this.sprite = sprite;
+        this._leftFlyTime = 0.7
+    }
+
+    isBroadcast() {
+        return false;
+    }
+
+    getColor() {
+        return 0xaee571;
```
Comment (Irving-cl) [inline web/site/js/vis/message.js:155]: Use a constant like `ACK_COLOR` or the color's name?

Comment (Irving-cl) [inline web/site/js/vis/message.js:155]: Use a constant like ACK_COLOR or the color's name?

Comment (simonlingoogle) [reply web/site/js/vis/message.js]: resolved

**Pull request merged**